jetpack.php

```diff
@@ -5,15 +5,15 @@
  * Plugin URI: https://jetpack.com
  * Description: Bring the power of the WordPress.com cloud to your self-hosted WordPress. Jetpack enables you to connect your blog to a WordPress.com account to use the powerful features normally only available to WordPress.com users.
  * Author: Automattic
- * Version: 8.9.1
+ * Version: 9.0
```

Comment:
We should put 9.0.1 here, as it's the actual version we are going to deploy

Reply:
Yeah I was thinking about it, but then when we will deploy 9.0.2, will we update this file as well?

Reply:
If someone uses a constant to pin to a different version, and they did `wp plugin list`, would it be the version value on line 8 here, or the version number from the version they've pinned to?

i.e. now that vip-go-mu-plugins includes more versions of Jetpack, should this file be treater as a "Jetpack Loader for VIP" plugin, and then have it's own version (`1.0.0`)?

Reply:
Good point. I tried it out and it shows version from this file, seems regardless of what version is loaded (be it as default or pinned). 

```
wp plugin list --allow-root
+-------------------------+----------+-----------+---------+
| name                    | status   | update    | version |
+-------------------------+----------+-----------+---------+
| jetpack                 | must-use | none      | 9.0     |
```

Reply:
> i.e. now that vip-go-mu-plugins includes more versions of Jetpack, should this file be treater as a "Jetpack Loader for VIP" plugin, and then have it's own version (1.0.0)?

We should also move the file to something like `jetpack-loader.php` to show it as jetpack-loader in the plugin list rather than jetpack, or?

Reply:
I have updated the version to be `9.0.1` for now, as maybe we would like to do the jetpack-loader change as a separate change. Maybe discuss more broadly?

Reply:
Agreed, should be handled outside of this PR. I just wanted to draw attention to the potential issue as I saw it and thought about it, and get it in your collective radar.

> We should also move the file to something like jetpack-loader.php to show it as jetpack-loader in the plugin list rather than jetpack, or?

Shouldn't be necessary to rename the file - just the "Plugin Name: Jetpack by WordPress.com" header, though if changing that name, it would likely then make sense to rename the file as well.

Reply:
@GaryJones I agree with you, I also raised this concern before. We could even just leave it as plugin `jetpack` but with no version. It's a pity that WordPress doesn't allow to set the version number dynamically...